tutor/specs/e2e/preview-courses.e2e.ts

```diff
@@ -16,38 +11,35 @@ test.describe('Preview Courses', () => {
             await page.click('testId=show-detail')
             await page.click('testId=create-preview')
         }
-    })
 
-    test('displays a collapsible side panel that creates a course', async ({ page }) => {
         await visitPage(page, '/courses')
         await page.click('testId=preview-course-card')
         await page.click('testId=preview-panel-create-course')
         await expect(page).toHaveSelector('testId=new-course-wizard')
         await expect(page).toHaveSelector('.select-dates')
+
     })
 
-    test('hides preview panel for non-preview courses', async ({ page }) => {
+    test('hides preview panel if the offering is not available for course creation', async ({ page }) => {
+        await findOrCreateTeacherAccount({ page, userName: 'previewteacher' })
```

Comment:
I think there might already be a `previewteacher` account setup on the backend.  If so we can call `loginAs` which is much faster

Reply:
It doesn't yet but that would be really nice to add, that would make this whole test very speedy